tweb/src/main/frontend/src/js/vue/components/RequestList.vue

```diff
@@ -129,6 +133,12 @@
         mounted() {
             this.sortBy = this.sorting[1].value;
             this.requestList = new RequestListModel(this.requests);
+
+			EventBus.$on("request-update", request => {
+				console.log("masterful update!!!!");
```

Comment:
Remove console.log

Reply:
I always do this at least once. Sorry, Steve.